rasa/core/utils.py

```diff
@@ -538,3 +539,13 @@ def _log_and_get_default_number_of_workers():
         f"no `RedisLockStore` endpoint configuration has been found."
     )
     return _log_and_get_default_number_of_workers()
+
+
+def create_sequence(table_name: Text) -> Sequence:
+    """Creates a sequence object"""
```

Comment:
let's add a full docstring which also describes the variables and return values.

Reply:
either here or in the column description: describe why we need the sequence